conn.go

```diff
@@ -25,25 +24,24 @@ func beginHandlerOpen(ch chan error, removeHandlers func()) func(ev *js.Object)
 
 // closeError allows a CloseEvent to be used as an error.
 type closeError struct {
-	*dom.CloseEvent
+	*js.Object
```

Comment:
Now that I think about it, maybe it's worth doing the same for message events? I.e., create a:

```
type messageEvent struct {
	*js.Object
	Data *js.Object `js:"data"`
}
```

Then do `c.ch <- &messageEvent{Object: event}` and `frame.Data`. I'm not quite sure if it's worth doing or not... Any thoughts on this?